src/catalogue.js

```diff
@@ -0,0 +1,119 @@
+import { first, pick, omit, get } from 'lodash';
+import * as xhr from './xhr';
+import Promise from 'bluebird';
+
+const BASIC_PROPS = ['title', 'summary', 'identifier', 'uri'];
+const REQUEST_DEFAULTS = {
+    types: ['attribute', 'metric', 'fact'],
+    paging: {
+        offset: 0,
+        limit: 'ALL'
+    }
+};
```

Comment:
We are paging all request now and so limit: 'ALL' doesn't make sense anymore as a default. Remove paging.